README.md

```diff
@@ -11,7 +11,28 @@ The API is contained in `CqlMigrator`, and works on directories of files ending
 
 For example, to apply all `.cql` files located in `/cql` in the classpath:
 
-    CqlMigrator migrator = new CqlMigrator();
+Configure the lock:
+
+    CassandraLockConfig lockConfig = CassandraLockConfig.builder()
+                    .withTimeout(Duration.standardSeconds(3))
+                    .withPollingInterval(Duration.millis(500))
+                    .withClientId("127.0.0.1")
+                    .withSimpleStrategyReplication(3)
+                    .build();
+                    
+Or
+
+    CassandraLockConfig lockConfig = CassandraLockConfig.builder()
+                    .withTimeout(Duration.standardSeconds(3))
+                    .withPollingInterval(Duration.millis(500))
+                    .withClientId("127.0.0.1")
+                    .withNetworkTopologyReplication("DC1", 3)
+                    .withNetworkTopologyReplication("DC2", 3)
+                    .build();
+
+Then:                    
```

Comment:
Everything has a default. It's just not mentioned in the README, we will add it.